ai_genomics/pipeline/README.md

```diff
@@ -29,10 +29,18 @@ aws s3 cp s3://ai-genomics/inputs/openalex/concepts.json inputs/openalex/concept
 
 To fetch other article category data we use to label the OpenAlex data.
 
-Run `python ai_genomics/analysis/openalex_definition.py` to implement the definitions and generate results. This includes printing stats in the console and saving relevant tables in `inputs/data`. This will also save figures to `outputs/figures`, the provisional ai genomics OpenAlex dataset to `outputs/ai_genomics_provisional_dataset.csv` and examples from the dataset to `outputs/openalex_examples.md`,
+Run `python ai_genomics/analysis/openalex_definition.py` to implement the definitions and generate results. This includes printing stats in the console and saving relevant tables in `inputs/data`. This will also save figures to `outputs/figures`, the provisional ai genomics OpenAlex dataset to `outputs/ai_genomics_provisional_dataset.csv` and examples from the dataset to `outputs/openalex_examples.md`.
```

Comment:
I know this is just a typo fix, but do you know what is meant by "implement the definitions" here?

Reply:
I think Juan means implementing the definition of AI and the definition of genomics to find the combined AI and genomics works.